* don't rebuild on install and check
* enable parallel tests
* comment out installing missing file
author: Miguel Dias Costa (National University of Singapore)
--- Makefile.orig	2021-05-16 16:29:50.000000000 +0900
+++ Makefile	2021-05-16 16:27:16.000000000 +0900
@@ -133,7 +133,7 @@
 manual:
 	(cd documentation/users/manual && python assemble_manual.py)
 
-install: all
+install: 
 ifdef INSTDIR
 	mkdir -p $(INSTDIR)/bin
 	install bin/*.x $(INSTDIR)/bin/
@@ -151,13 +151,13 @@
 # install cannot work on a whole directory
 	cp -rf examples  $(INSTDIR)/share/BerkeleyGW/
 	cp -rf testsuite $(INSTDIR)/share/BerkeleyGW/
-	install manual.html $(INSTDIR)/share/BerkeleyGW/
+	#install manual.html $(INSTDIR)/share/BerkeleyGW/
 else
 	$(error Error: Please define installation prefix INSTDIR via 'make install INSTDIR='.)
 endif
 
-check: all
-	cd testsuite && $(MAKE) check
+check:
+	cd testsuite && $(MAKE) check && $(MAKE) check-parallel
 
 check-save: all
 	cd testsuite && $(MAKE) check-save
